gui/control-panel.lua

```diff
@@ -910,7 +773,9 @@ function ControlPanel:init()
                 'Autostart',
             },
             on_select=self:callback('set_page'),
-            get_cur_page=function() return self.subviews.pages:getSelected() end
+            get_cur_page=function() return self.subviews.pages:getSelected() end,
+            key='CUSTOM_ALT_T',
+            key_back='CUSTOM_ALT_R',
```

Comment:
Could we use Ctrl instead of Alt? Alt keys have a higher chance of system conflict, and I'd like to standardize on Ctrl-T to mean "next tab". Ctrl-R would need to be changed to something else. It would be nice if we had access to Ctrl-Shift-T, but we don't : /. Maybe Ctrl-Y?